features/share/graphql/creating-our-uni.md

```diff
@@ -0,0 +1,46 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# Creating Our Universal Application
+
+## Pre-requisites
+
+* [Vendia Share CLI](https://vendia.net/docs/share/cli)
+
+## Deploying the Uni
+
+This example will create a Uni to store attributes regarding product inventory.
+
+If you are not already logged in to the share service you do so by running [`share login`](https://vendia.net/docs/share/cli/commands/login):
+
+```bash
+share login
+```
+
+The `share uni create` command can be used to deploy our Uni. You will need to copy the file `registration.json.sample` to `registration.json`. Pick a unique `name` for your uni that begins with `test-` - by default all unis share a common namespace so here is your chance to get creative. Update the `userId` attribute of each node to reflect your Vendia Share email address.
```

Comment:
Why does the Uni need to begin with `test-`? Also what do you mean by "by default all unis share a common namespace so here is your chance to get creative."?

Reply:
We don't really define what a namespace is in our docs. This will be changed. I've also created a task to address the Uni naming recommendation.